src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -73,6 +74,9 @@ async function boot(options?: Partial<WebAssemblyStartOptions>): Promise<void> {
     }
   };
 
+  if (document.getElementById ('blazor-default-loading'))
+    WebAssemblyProgressReporter.init();
```

Comment:
We tend to prefer brackets (even if they aren't strictly required) for code style:

```suggestion
  if (document.getElementById('blazor-default-loading')) {
    WebAssemblyProgressReporter.init();
  }
```